node/service/src/chain_spec/test_spec.rs

```diff
@@ -39,7 +39,7 @@ pub fn staking_spec(para_id: ParaId) -> ChainSpec {
 	.with_properties(
 		serde_json::from_str("{\"tokenDecimals\": 18}").expect("Provided valid json map"),
 	)
-	.with_genesis_config(testnet_genesis(
+	.with_genesis_config(moonbase_runtime::genesis_config_preset::testnet_genesis(
```

Comment:
```suggestion
	.with_genesis_config_preset_name(sp_genesis_builder::DEV_RUNTIME_PRESET)
```

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3138/commits/6ef78e33b5f89ecdb53d07da071a3f7de6f1091a